bftengine/src/bftengine/RequestHandler.cpp

```diff
@@ -56,6 +56,14 @@ void RequestHandler::execute(IRequestsHandler::ExecutionRequestsQueue& requests,
                              concordUtils::SpanWrapper& parent_span) {
   bool has_pruning_request = false;
   for (auto& req : requests) {
+    LOG_INFO(GL,
```

Comment:
I suggest moving this to DEBUG (this will be print on every request execution, in a real work load we may have dozens - hundreds requests in a batch

Reply:
Done